src/main/java/com/Elessar/app/Main.java

```diff
@@ -9,7 +14,27 @@
     public static void main(String[] args){
         final MyServer server = new MyServer("localhost", 9000);
         server.run();
+        testProtoBuf();
+
     }
-}
 
+    private static void testProtoBuf() {
+        Person.Builder sendPersonData = Person.newBuilder().setId(123).setName("DIBL").setEmail("abc@gmail.com");
```

Comment:
You can (and probably should) add "final" to a bunch of variables in this function.